book/website/community-handbook/community-calls/community-calls-steering-committee.md

```diff
@@ -0,0 +1,15 @@
+(ch-calls-steering-committee)=
+# Steering Committee Meetings 
+
+```{warning} This page is under review and will be updated with more details soon.
+```
+## Steering Committee Meetings
+In 2025, Steering Committee Meetings will be hosted regularly to formalise Constitution Level governance.
```

Comment:
I think it might be better to talk about the purpose of the meetings without this year as an exception.